Makefile

```diff
@@ -1,5 +1,5 @@
 .PHONY: default
-default: deps checkgofmt vet predeclared staticcheck ineffassign golint test
+default: deps checkgofmt vet predeclared staticcheck ineffassign errcheck golint golint test
```

Comment:
I'll accept these changes, many of which were made to satisfy the linter. But don't be surprised when I remove `golint` from this list in the future.

Reply:
That's totally fine - I'd argue to keep it until there's something that annoys you, but I get it. I personally like that it only lets me do things in a certain way, but that's not everyone's cup of tea.